dev-lang/nqp/nqp-2023.02-r1.ebuild

```diff
@@ -5,35 +5,33 @@ EAPI=8
 
 inherit java-pkg-opt-2 multibuild
 
-if [[ ${PV} == "9999" ]]; then
-	EGIT_REPO_URI="https://github.com/perl6/${PN}.git"
-	inherit git-r3
-else
-	SRC_URI="https://github.com/perl6/${PN}/releases/download/${PV}/${P}.tar.gz"
-	KEYWORDS="~amd64 ~x86"
-fi
-
 DESCRIPTION="Not Quite Perl, a Raku bootstrapping compiler"
-HOMEPAGE="https://rakudo.org"
-
+HOMEPAGE="https://rakudo.org
+	https://github.com/Raku/nqp"
+SRC_URI="https://github.com/Raku/${PN}/releases/download/${PV}/${P}.tar.gz"
 LICENSE="Artistic-2"
 SLOT="0"
-IUSE="doc clang java +moar test"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc java +moar test"
 RESTRICT="!test? ( test )"
 REQUIRED_USE="|| ( java moar )"
 
-CDEPEND="java? (
+CDEPEND="
+	java? (
 		dev-java/asm:9
 		dev-java/jna:4
 	)
-	moar? ( ~dev-lang/moarvm-${PV}[clang=] )"
-RDEPEND="${CDEPEND}
-	java? ( >=virtual/jre-11 )"
+	moar? ( ~dev-lang/moarvm-${PV} )
+"
+RDEPEND="
+	${CDEPEND}
+	java? ( >=virtual/jre-11:* )
+"
 DEPEND="${CDEPEND}"
```

Comment:
```
DEPEND="
	${CDEPEND}
	java? ( >=virtual/jdk-11:* )
"
```

Reply:
Hmm? Why do you tell me to roll back the change you just requested?